src/ui/Toast.tsx

```diff
@@ -87,35 +66,43 @@ const VariantToStylesMap = {
 
 type Props = {
   variant: Values<typeof ToastVariants>;
-  header: ReactNode;
   body: string;
   visible?: boolean;
   onClose?: () => void;
 };
 
-const Toast = ({ variant, visible, header, body, onClose }: Props) => {
+const Toast = ({ variant, visible, body, onClose }: Props) => {
+  const icon = useMemo(() => {
+    const icons = {
+      [ToastVariants.PRIMARY]: Icons.PLUS_CIRCLE,
```

Comment:
This should probably be an info icon? With a question mark?
Maybe we only need 3 toast variants: success, danger and warning?

Reply:
I'm not even sure danger and warning are used at the moment, the only variant in use I could find was success